challenges/visual_referee_challenge.tex

```diff
@@ -1,3 +1,163 @@
+Building on the Visual Referee Challenge from 2022, this year's In-game Visual Referee Challenge asks the robots to detect visual referee signs during a regular match when a whistle has been blown. The normal game play is not influenced by this challenge.
+
+This technical challenge tests a robot's ability to identify three categories of hand signals during a match:
+\begin{enumerate}
+    \item Static hand signals with one hand.
+    \item Static hand signals with two hands.
+    \item Dynamic (motion) hand signals with one or two hands.
+\end{enumerate}
+
+The intent of this challenge is to choose a \emph{subset} of potential referee calls in SPL matches and test ability of a team to recognize different types of hand signals in preparation for adoption in RoboCup matches.
+
+\subsection{Challenge setup}
+
+As this is in-game challenge, the challenge is executed during all preliminary matches of the main competition. All usual rules apply and teams are free to participate or not.
+
+During \textit{Set} and anytime during \textit{Playing} when the head referee whistles (Kick-off, goal, half ends), the challenged team (can be both teams) have to look to the T-junction opposite to the GameController operator. At the T-junction of the center field line a challenge assistant will show within \qty{5}{\second} after the whistle a random hand-signal and direction according to the list below for \qty{10}{\second}. The hand-signals and directions are randomly chosen but within the first four whistles there have to be hand-signals from all three classes. The same hand-signal may be chosen twice (with different directions). The challenge assistant is wearing red gloves to distinguish him from the other referees. In case the head referee has to stand on the T-junction, he will stand close to the challenge assistant and do his referee job from that close by position.
+
+The challenge assistant must wear referee cloth and red gloves. The purpose of this clothing is to clearly distinguish the challenge assistant and its hands from the other referees and from the background people. 
+
+The description of this challenge and hand-signals are described based on the viewpoint of the challenge assistant. In these descriptions from the perspective of the head referee the ``red team'' is defined as playing from left-to-right, and the ``blue team'' as playing from right-to-left. The use of colors for identifying teams is used to give equivalence to the head referee calls during the match. Blue and red have to be replaced by the actual colors of that particular match.
+
+The challenged team represented by the robot with the lowest number reports its evaluation of the particular hand-signal and direction using very loud sound output, repeating the result four times and sends a message with the decision over Wifi. \textbf{Details about this will come.} Only Messages with audio output and message sent within \qty{15}{second} after the signalling started, will be accepted.
```

Comment:
Only Wifi, put into GameController return message.

Reply:
Think about message budget for other challenge

Reply:
Just take the first incoming message by a team, otherwise it is the one with the lowest number

Reply:
a second whistle within 30s it has to be ignored

Reply:
robot should send whistle, decision, team and timestamp